indigo_app/templates/indigo_app/place/_work_detail.html

```diff
@@ -84,3 +84,33 @@
     <div class="tab-pane" id="work-{{ work.pk }}-tasks"></div>
   {% endif %}
 </div>
+
+{% if perms.indigo_api.change_work or perms.indigo_api.bulk_add_work %}
+  <div class="border-top p-2 {% if work.work_in_progress %}bg-work-in-progress-light{% else %}bg-light{% endif %}">
+    {% if perms.indigo_api.change_work %}
+      <button
+          id="edit_work_button"
+          hx-get="{% url 'work_edit_offcanvas' work.frbr_uri %}"
+          hx-target="#offCanvasEditWorkForm"
+          hx-swap="innerHTML"
+          class="btn btn-outline-primary btn-sm"
+          type="button"
+          data-bs-toggle="offcanvas"
+          data-bs-target="#offcanvasEditWork"
+          aria-controls="offcanvasEditWork">
+        <span>{% trans 'Edit' %}</span>
+      </button>
+    {% endif %}
+    {% if perms.indigo_api.bulk_add_work %}
+      {% if work.work_in_progress %}
+        <button type="button" class="btn btn-success btn-sm" hx-post="{% url 'work_approve' frbr_uri=work.frbr_uri %}">
```

Comment:
so the card content gets re-rendered completely, right? How does that work? Wouldn't than need `hx-target="..."` or am I missing something?

Reply:
the ApproveWorkView returns a redirect, and the card is reloaded, from what I can tell.
I could tell because when I didn't do that the whole page loaded inside that card again — it was amazing 😂